rpg_api/tests/pytest/unit/test_unit_test_utils.py

```diff
@@ -0,0 +1,96 @@
+import pytest
+import uuid
+from httpx import Response
+from unittest.mock import patch
+from rpg_api.tests.pytest.test_utils import get_user_header, get_data
+import jwt
+
+
+def test_get_user_header_valid_user_id() -> None:
+    """
+    Test that get_user_header returns the correct header,
+    when provided with a valid user_id.
+    """
+    user_id = uuid.uuid4()
+    with (
+        patch("rpg_api.utils.dtos.TokenData") as mock_token_data,
+        patch(
+            "rpg_api.web.api.postgres.auth.auth_utils.create_access_token",
+            return_value="test_token",
+        ),
+    ):
+        header = get_user_header(user_id)
+        mock_token_data.assert_called_with(user_id=str(user_id))
+        assert header == {"Authorization": "Bearer test_token"}
```

Comment:
Nice, but I think the layouts of some of the tests can be improved a bit, so that they follow the triple A pattern.
```suggestion
def test_get_user_header_with_valid_user_id() -> None:
    """Test get_user_header with a valid user_id should return correct header."""
    # arrange
    user_id = uuid.uuid4()
    expected_token = "test_token"
    expected_header = {"Authorization": f"Bearer {expected_token}"}

	# act
    with patch("rpg_api.utils.dtos.TokenData") as mock_token_data, patch(
        "rpg_api.web.api.postgres.auth.auth_utils.create_access_token",
        return_value=expected_token,
    ):
        header = get_user_header(user_id)

		# assert
        mock_token_data.assert_called_with(user_id=str(user_id))
        assert header == expected_header
```

ignore the arrange, act, assert comments.